src/utils/Sort.ts

```diff
@@ -0,0 +1,5 @@
+export enum Sort {
```

Comment:
Consider renaming the enum from 'Sort' to 'Filter' to better reflect its purpose in filtering todos by status.